### PR TITLE
C1 beshoy profile gui

### PR DESCRIPTION
**Pull request by BeshoyAtef**
@HebaElSobky , @abdelrahmanMaged  , @mohamedahmedawad93 

Please review my code and check if their is any comments

### CODE REVIEW

**Comment by BeshoyAtef** [comment]
#214

**Comment by HebaElSobky** [comment]
@BeshoSmile please when edit right a placeholder in the textbox of the date of birth to choose a format of ISO

**Comment by BeshoyAtef** [comment]
@HebaElSobky

**Comment by HebaElSobky** [comment]
@BeshoSmile please add the placeholder of the textbox of the date that says "please choose the ISO formate i just checked and it's not there !!

**Comment by BeshoyAtef** [comment]
ok ha5od al info box mn ur branch copy paste

**Comment by HebaElSobky** [comment]
Okaay

Sent from my iPhone,
Heba A. El-Sobky :)

**Comment by HebaElSobky** [comment]
@BeshoSmile take it and tell me to check again

File: project/tager/tager_www/views.py [hunk 1]
```diff
@@ -269,7 +269,7 @@ def view_post(request):
     user = request.user
     post_id = request.GET['post']
     post = Post.objects.get(id=post_id)
-    post_can_be_wished = user.add_to_wish_list(post_id)
+    # post_can_be_wished = user.add_to_wish_list(post_id)
     test_post = Post.objects.get(id = post_id)
     test_post.post_state
     subchannel1 = test_post.subchannel_id
```
Comment (mohamedahmedawad93) [inline project/tager/tager_www/views.py:282]: @BeshoSmile this should be subchannel not subchannel_id according to the convention

Comment (BeshoyAtef) [inline project/tager/tager_www/views.py:282]: this is not my method asln , my story is #214

Comment (mohamedahmedawad93) [inline project/tager/tager_www/views.py:282]: @BeshoSmile tayeb taba3 meen?? 3alashan hatedy errorrs

Comment (abdelrahmanMaged) [inline project/tager/tager_www/views.py:282]: it is mine y awad :D

**Comment by BeshoyAtef** [comment]
@AbdelrahmanAyad

**Comment by abdelrahmanMaged** [comment]
regarding this branch i can not review it as n happy already started
fhmny??

**Comment by BeshoyAtef** [comment]
happy btreview 7agatha anha zai am hya mesh senario

On Mon, Apr 29, 2013 at 10:19 PM, abdelrahmanMaged <notifications@github.com

> wrote:
> 
> regarding this branch i can not review it as n happy already started
> fhmny??
>

**Comment by abdelrahmanMaged** [comment]
oki execuse i isunderstood d :(

**Comment by BeshoyAtef** [comment]
:+1:

**Comment by abdelrahmanMaged** [comment]
@BeshoSmile update the branch so it can be green :D

**Comment by BeshoyAtef** [comment]
mashy bs brdo reviewho w hwa keda w  ana hzbto b3deen

On Mon, Apr 29, 2013 at 10:30 PM, abdelrahmanMaged <notifications@github.com

**Comment by abdelrahmanMaged** [comment]
bs kda kd ait will not be merged :D
ole nta 3aml ay backend wla hwa ana babos 3l layout wkda?? 3shan ab2a fahm
b review a bs :D

**Comment by BeshoyAtef** [comment]
only frnd end !!

On Mon, Apr 29, 2013 at 10:34 PM, abdelrahmanMaged <notifications@github.com

> wrote:
> 
> bs kda kd ait will not be merged :D
> ole nta 3aml ay backend wla hwa ana babos 3l layout wkda?? 3shan ab2a fahm
> b review a bs :D
>

**Comment by BeshoyAtef** [comment]
front*

**Comment by abdelrahmanMaged** [comment]
verified bs remeber 7war n l 7eta l maktob 3aleha wish list d will be
button eshta?

**Comment by BeshoyAtef** [comment]
eshta 
@Tharwat7

**Comment by BeshoyAtef** [comment]
@Tharwat7 all my labels are verified please merge me

**Comment by HalaKamal** [comment]
when your diagram is finished tage me in both pull request and issue to review @BeshoSmile

**Comment by HalaKamal** [comment]
@BeshoSmile  i cant see them tell me where

**Comment by BeshoyAtef** [comment]
<http://www.skillpages.com/i/ftuf/emailinvitelanding?user=88470b98-4248-e011-889f
-c965259bf186&signedParameters=ebrgo72E1Nxm7RVE05Hski9egwQ2KwFqIDWQQt_qqYo.eyJlbW
FpbCI6InJlcGx5K2ktMTM3MzY2ODktYWZlMDI3NDcwODYwMDVhM2M3ZDJjZWVkNjJmYTM3ODRmYzFhYzI
xMC0zNTE1Mjc4QHJlcGx5LmdpdGh1Yi5jb20iLCJmcm9tIjoiODg0NzBiOTgtNDI0OC1lMDExLTg4OWYt
Yzk2NTI1OWJmMTg2IiwidG8iOiJmMzQyMWQ1Yy0zNTE2LTU0ZTYtODRmNi00MjEzM2M1YWQ5NzQiLCJpc
3N1ZWRfYXQiOiIxMzY5ODQ1NDA2IiwidmVyIjoiMSIsInAiOiIzIn0&emailMessageId=5629f5ef-7d
c8-e211-a4c9-c701ce909180&emailType=Invite&sdate=201305291636&utm_source=EmailInv
ite&utm_medium=Email&sendercc=EG&emailLinkType=Logo>

Hi,

Beshoy Atef wants to connect with you.

<http://www.skillpages.com/i/ftuf/emailinvitelanding?user=88470b98-4248-e011-889f
-c965259bf186&signedParameters=ebrgo72E1Nxm7RVE05Hski9egwQ2KwFqIDWQQt_qqYo.eyJlbW
FpbCI6InJlcGx5K2ktMTM3MzY2ODktYWZlMDI3NDcwODYwMDVhM2M3ZDJjZWVkNjJmYTM3ODRmYzFhYzI
xMC0zNTE1Mjc4QHJlcGx5LmdpdGh1Yi5jb20iLCJmcm9tIjoiODg0NzBiOTgtNDI0OC1lMDExLTg4OWYt
Yzk2NTI1OWJmMTg2IiwidG8iOiJmMzQyMWQ1Yy0zNTE2LTU0ZTYtODRmNi00MjEzM2M1YWQ5NzQiLCJpc
3N1ZWRfYXQiOiIxMzY5ODQ1NDA2IiwidmVyIjoiMSIsInAiOiIzIn0&emailMessageId=5629f5ef-7d
c8-e211-a4c9-c701ce909180&emailType=Invite&sdate=201305291636&utm_source=EmailInv
ite&utm_medium=Email&sendercc=EG&emailLinkType=ProfileThumb>

Beshoy Atef
<http://www.skillpages.com/i/ftuf/emailinvitelanding?user=88470b98-4248-e011-889f
-c965259bf186&signedParameters=ebrgo72E1Nxm7RVE05Hski9egwQ2KwFqIDWQQt_qqYo.eyJlbW
FpbCI6InJlcGx5K2ktMTM3MzY2ODktYWZlMDI3NDcwODYwMDVhM2M3ZDJjZWVkNjJmYTM3ODRmYzFhYzI
xMC0zNTE1Mjc4QHJlcGx5LmdpdGh1Yi5jb20iLCJmcm9tIjoiODg0NzBiOTgtNDI0OC1lMDExLTg4OWYt
Yzk2NTI1OWJmMTg2IiwidG8iOiJmMzQyMWQ1Yy0zNTE2LTU0ZTYtODRmNi00MjEzM2M1YWQ5NzQiLCJpc
3N1ZWRfYXQiOiIxMzY5ODQ1NDA2IiwidmVyIjoiMSIsInAiOiIzIn0&emailMessageId=5629f5ef-7d
c8-e211-a4c9-c701ce909180&emailType=Invite&sdate=201305291636&utm_source=EmailInv
ite&utm_medium=Email&sendercc=EG&emailLinkType=ProfileName>

Invite sent: 29 May, 2013

Continue
<http://www.skillpages.com/i/ftuf/emailinvitelanding?user=88470b98-4248-e011-889f
-c965259bf186&signedParameters=ebrgo72E1Nxm7RVE05Hski9egwQ2KwFqIDWQQt_qqYo.eyJlbW
FpbCI6InJlcGx5K2ktMTM3MzY2ODktYWZlMDI3NDcwODYwMDVhM2M3ZDJjZWVkNjJmYTM3ODRmYzFhYzI
xMC0zNTE1Mjc4QHJlcGx5LmdpdGh1Yi5jb20iLCJmcm9tIjoiODg0NzBiOTgtNDI0OC1lMDExLTg4OWYt
Yzk2NTI1OWJmMTg2IiwidG8iOiJmMzQyMWQ1Yy0zNTE2LTU0ZTYtODRmNi00MjEzM2M1YWQ5NzQiLCJpc
3N1ZWRfYXQiOiIxMzY5ODQ1NDA2IiwidmVyIjoiMSIsInAiOiIzIn0&cnId=88470b98-4248-e011-88
9f-c965259bf186&emailMessageId=5629f5ef-7dc8-e211-a4c9-c701ce909180&emailType=Inv
ite&sdate=201305291636&utm_source=EmailInvite&utm_medium=Email&sendercc=EG&emailL
inkType=MainCallToAction>

reply+i-13736689-afe02747086005a3c7d2ceed62fa3784fc1ac210-3515278@reply.github.co
m was invited to join SkillPages by Beshoy Atef. To stop receiving emails from
SkillPages click here
<http://www.skillpages.com/i/account/unsubscribe?signedParameters=Q3f1OUm5kdqU4xy
7w4t5qBbEE8z1h92c77mcXJhobhI.eyJlbWFpbCI6InJlcGx5K2ktMTM3MzY2ODktYWZlMDI3NDcwODYw
MDVhM2M3ZDJjZWVkNjJmYTM3ODRmYzFhYzIxMC0zNTE1Mjc4QHJlcGx5LmdpdGh1Yi5jb20iLCJpc3N1Z
WRfYXQiOiIxMzY5ODQ1NDA2IiwidmVyIjoiMSIsInAiOiIwIn0&emailMessageId=5629f5ef-7dc8-e
211-a4c9-c701ce909180&emailType=Invite&sdate=201305291636&utm_source=EmailInvite&
utm_medium=Email&sendercc=EG&emailLinkType=Unsubscribe> .

© 2013 SkillPages, Blackrock Business Park, Dublin, Ireland and 228 Hamilton
Avenue, 3rd Floor, Palo Alto, CA 94301.

**Comment by BeshoyAtef** [comment]
<http://www.skillpages.com/i/ftuf/emailinvitelanding?user=88470b98-4248-e011-889f
-c965259bf186&signedParameters=3M8dRSYcAx4vkuUCzZRbgtwZGSxDuJj44RobydlKo-Y.eyJlbW
FpbCI6InJlcGx5K2ktMTM3MzY2ODktYWZlMDI3NDcwODYwMDVhM2M3ZDJjZWVkNjJmYTM3ODRmYzFhYzI
xMC0zNTE1Mjc4QHJlcGx5LmdpdGh1Yi5jb20iLCJmcm9tIjoiODg0NzBiOTgtNDI0OC1lMDExLTg4OWYt
Yzk2NTI1OWJmMTg2IiwidG8iOiJmMzQyMWQ1Yy0zNTE2LTU0ZTYtODRmNi00MjEzM2M1YWQ5NzQiLCJpc
3N1ZWRfYXQiOiIxMzcwMzkxODMxIiwidmVyIjoiMSIsInAiOiIzIn0&emailMessageId=0ac9fa14-73
cd-e211-be08-d229661189ee&emailType=InviteFirstReminder&sdate=201306050023&utm_so
urce=EmailInviteFirstReminder&utm_medium=Email&sendercc=EG&emailLinkType=Logo>

Hi,

Beshoy is waiting for you to accept this pending invitation.

<http://www.skillpages.com/i/ftuf/emailinvitelanding?user=88470b98-4248-e011-889f
-c965259bf186&signedParameters=3M8dRSYcAx4vkuUCzZRbgtwZGSxDuJj44RobydlKo-Y.eyJlbW
FpbCI6InJlcGx5K2ktMTM3MzY2ODktYWZlMDI3NDcwODYwMDVhM2M3ZDJjZWVkNjJmYTM3ODRmYzFhYzI
xMC0zNTE1Mjc4QHJlcGx5LmdpdGh1Yi5jb20iLCJmcm9tIjoiODg0NzBiOTgtNDI0OC1lMDExLTg4OWYt
Yzk2NTI1OWJmMTg2IiwidG8iOiJmMzQyMWQ1Yy0zNTE2LTU0ZTYtODRmNi00MjEzM2M1YWQ5NzQiLCJpc
3N1ZWRfYXQiOiIxMzcwMzkxODMxIiwidmVyIjoiMSIsInAiOiIzIn0&emailMessageId=0ac9fa14-73
cd-e211-be08-d229661189ee&emailType=InviteFirstReminder&sdate=201306050023&utm_so
urce=EmailInviteFirstReminder&utm_medium=Email&sendercc=EG&emailLinkType=ProfileT
humb>

Beshoy Atef
<http://www.skillpages.com/i/ftuf/emailinvitelanding?user=88470b98-4248-e011-889f
-c965259bf186&signedParameters=3M8dRSYcAx4vkuUCzZRbgtwZGSxDuJj44RobydlKo-Y.eyJlbW
FpbCI6InJlcGx5K2ktMTM3MzY2ODktYWZlMDI3NDcwODYwMDVhM2M3ZDJjZWVkNjJmYTM3ODRmYzFhYzI
xMC0zNTE1Mjc4QHJlcGx5LmdpdGh1Yi5jb20iLCJmcm9tIjoiODg0NzBiOTgtNDI0OC1lMDExLTg4OWYt
Yzk2NTI1OWJmMTg2IiwidG8iOiJmMzQyMWQ1Yy0zNTE2LTU0ZTYtODRmNi00MjEzM2M1YWQ5NzQiLCJpc
3N1ZWRfYXQiOiIxMzcwMzkxODMxIiwidmVyIjoiMSIsInAiOiIzIn0&emailMessageId=0ac9fa14-73
cd-e211-be08-d229661189ee&emailType=InviteFirstReminder&sdate=201306050023&utm_so
urce=EmailInviteFirstReminder&utm_medium=Email&sendercc=EG&emailLinkType=ProfileN
ame>

Invite sent: 29 May, 2013

Continue
<http://www.skillpages.com/i/ftuf/emailinvitelanding?user=88470b98-4248-e011-889f
-c965259bf186&signedParameters=3M8dRSYcAx4vkuUCzZRbgtwZGSxDuJj44RobydlKo-Y.eyJlbW
FpbCI6InJlcGx5K2ktMTM3MzY2ODktYWZlMDI3NDcwODYwMDVhM2M3ZDJjZWVkNjJmYTM3ODRmYzFhYzI
xMC0zNTE1Mjc4QHJlcGx5LmdpdGh1Yi5jb20iLCJmcm9tIjoiODg0NzBiOTgtNDI0OC1lMDExLTg4OWYt
Yzk2NTI1OWJmMTg2IiwidG8iOiJmMzQyMWQ1Yy0zNTE2LTU0ZTYtODRmNi00MjEzM2M1YWQ5NzQiLCJpc
3N1ZWRfYXQiOiIxMzcwMzkxODMxIiwidmVyIjoiMSIsInAiOiIzIn0&cnId=88470b98-4248-e011-88
9f-c965259bf186&emailMessageId=0ac9fa14-73cd-e211-be08-d229661189ee&emailType=Inv
iteFirstReminder&sdate=201306050023&utm_source=EmailInviteFirstReminder&utm_mediu
m=Email&sendercc=EG&emailLinkType=MainCallToAction>

reply+i-13736689-afe02747086005a3c7d2ceed62fa3784fc1ac210-3515278@reply.github.co
m was invited to join SkillPages by Beshoy Atef. To stop receiving emails from
SkillPages click here
<http://www.skillpages.com/i/account/unsubscribe?signedParameters=BFMVEZjGlYv_yOy
1vVLZSL8j9Fx0hZ_0S-JhtPLRyWM.eyJlbWFpbCI6InJlcGx5K2ktMTM3MzY2ODktYWZlMDI3NDcwODYw
MDVhM2M3ZDJjZWVkNjJmYTM3ODRmYzFhYzIxMC0zNTE1Mjc4QHJlcGx5LmdpdGh1Yi5jb20iLCJpc3N1Z
WRfYXQiOiIxMzcwMzkxODMxIiwidmVyIjoiMSIsInAiOiIwIn0&emailMessageId=0ac9fa14-73cd-e
211-be08-d229661189ee&emailType=InviteFirstReminder&sdate=201306050023&utm_source
=EmailInviteFirstReminder&utm_medium=Email&sendercc=EG&emailLinkType=Unsubscribe>
.

© 2013 SkillPages, Blackrock Business Park, Dublin, Ireland and 228 Hamilton
Avenue, 3rd Floor, Palo Alto, CA 94301.

**Comment by BeshoyAtef** [comment]
<http://www.skillpages.com/i/ftux/emailinvitelanding?signedParameters=pLKG3XZ1DO8
72U3jnvVCUXvBNOOCIr4vm6Nrqeuuah0.eyJlbWFpbCI6InJlcGx5K2ktMTM3MzY2ODktYWZlMDI3NDcw
ODYwMDVhM2M3ZDJjZWVkNjJmYTM3ODRmYzFhYzIxMC0zNTE1Mjc4QHJlcGx5LmdpdGh1Yi5jb20iLCJmc
m9tIjoiODg0NzBiOTgtNDI0OC1lMDExLTg4OWYtYzk2NTI1OWJmMTg2IiwidG8iOiJmMzQyMWQ1Yy0zNT
E2LTU0ZTYtODRmNi00MjEzM2M1YWQ5NzQiLCJpc3N1ZWRfYXQiOiIxMzcwOTk2ODIxIiwidmVyIjoiMSI
sInAiOiIzIn0&emailMessageId=a0672e48-f3d2-e211-884e-85d1397596ad&emailType=Invite
SecondReminder&sdate=201306120027&utm_source=EmailInviteSecondReminder&utm_medium
=Email&sendercc=EG&emailLinkType=Logo>

Hi,

Beshoy Atef is still waiting to connect with you.

<http://www.skillpages.com/i/ftux/emailinvitelanding?signedParameters=ftux/emaili
nvitelanding?signedParameters=pLKG3XZ1DO872U3jnvVCUXvBNOOCIr4vm6Nrqeuuah0.eyJlbWF
pbCI6InJlcGx5K2ktMTM3MzY2ODktYWZlMDI3NDcwODYwMDVhM2M3ZDJjZWVkNjJmYTM3ODRmYzFhYzIx
MC0zNTE1Mjc4QHJlcGx5LmdpdGh1Yi5jb20iLCJmcm9tIjoiODg0NzBiOTgtNDI0OC1lMDExLTg4OWYtY
zk2NTI1OWJmMTg2IiwidG8iOiJmMzQyMWQ1Yy0zNTE2LTU0ZTYtODRmNi00MjEzM2M1YWQ5NzQiLCJpc3
N1ZWRfYXQiOiIxMzcwOTk2ODIxIiwidmVyIjoiMSIsInAiOiIzIn0&emailMessageId=a0672e48-f3d
2-e211-884e-85d1397596ad&emailType=InviteSecondReminder&sdate=201306120027&utm_so
urce=EmailInviteSecondReminder&utm_medium=Email&sendercc=EG&emailLinkType=Profile
Thumb>

Beshoy Atef
<http://www.skillpages.com/i/ftux/emailinvitelanding?signedParameters=ftux/emaili
nvitelanding?signedParameters=pLKG3XZ1DO872U3jnvVCUXvBNOOCIr4vm6Nrqeuuah0.eyJlbWF
pbCI6InJlcGx5K2ktMTM3MzY2ODktYWZlMDI3NDcwODYwMDVhM2M3ZDJjZWVkNjJmYTM3ODRmYzFhYzIx
MC0zNTE1Mjc4QHJlcGx5LmdpdGh1Yi5jb20iLCJmcm9tIjoiODg0NzBiOTgtNDI0OC1lMDExLTg4OWYtY
zk2NTI1OWJmMTg2IiwidG8iOiJmMzQyMWQ1Yy0zNTE2LTU0ZTYtODRmNi00MjEzM2M1YWQ5NzQiLCJpc3
N1ZWRfYXQiOiIxMzcwOTk2ODIxIiwidmVyIjoiMSIsInAiOiIzIn0&emailMessageId=a0672e48-f3d
2-e211-884e-85d1397596ad&emailType=InviteSecondReminder&sdate=201306120027&utm_so
urce=EmailInviteSecondReminder&utm_medium=Email&sendercc=EG&emailLinkType=Profile
Name>

Invite sent: 29 May, 2013

Continue
<http://www.skillpages.com/i/ftux/emailinvitelanding?signedParameters=pLKG3XZ1DO8
72U3jnvVCUXvBNOOCIr4vm6Nrqeuuah0.eyJlbWFpbCI6InJlcGx5K2ktMTM3MzY2ODktYWZlMDI3NDcw
ODYwMDVhM2M3ZDJjZWVkNjJmYTM3ODRmYzFhYzIxMC0zNTE1Mjc4QHJlcGx5LmdpdGh1Yi5jb20iLCJmc
m9tIjoiODg0NzBiOTgtNDI0OC1lMDExLTg4OWYtYzk2NTI1OWJmMTg2IiwidG8iOiJmMzQyMWQ1Yy0zNT
E2LTU0ZTYtODRmNi00MjEzM2M1YWQ5NzQiLCJpc3N1ZWRfYXQiOiIxMzcwOTk2ODIxIiwidmVyIjoiMSI
sInAiOiIzIn0&emailMessageId=a0672e48-f3d2-e211-884e-85d1397596ad&emailType=Invite
SecondReminder&sdate=201306120027&utm_source=EmailInviteSecondReminder&utm_medium
=Email&sendercc=EG&emailLinkType=MainCallToAction>

reply+i-13736689-afe02747086005a3c7d2ceed62fa3784fc1ac210-3515278@reply.github.co
m was invited to join SkillPages by Beshoy Atef. To stop receiving emails from
SkillPages click here
<http://www.skillpages.com/i/account/unsubscribe?signedParameters=O5BKnZQbcLIBK_i
roVvMFOCHqWbRITt4MTVYy6xfYpc.eyJlbWFpbCI6InJlcGx5K2ktMTM3MzY2ODktYWZlMDI3NDcwODYw
MDVhM2M3ZDJjZWVkNjJmYTM3ODRmYzFhYzIxMC0zNTE1Mjc4QHJlcGx5LmdpdGh1Yi5jb20iLCJpc3N1Z
WRfYXQiOiIxMzcwOTk2ODIxIiwidmVyIjoiMSIsInAiOiIwIn0&emailMessageId=a0672e48-f3d2-e
211-884e-85d1397596ad&emailType=InviteSecondReminder&sdate=201306120027&utm_sourc
e=EmailInviteSecondReminder&utm_medium=Email&sendercc=EG&emailLinkType=Unsubscrib
e> .

© 2013 SkillPages, Blackrock Business Park, Dublin, Ireland and 228 Hamilton
Avenue, 3rd Floor, Palo Alto, CA 94301.

**Pull request merged**